indico/modules/rb_new/client/js/common/timeline/TimelineLegend.module.scss

```diff
@@ -18,44 +18,49 @@
 @import 'rb_new:styles/palette';
 @import 'rb_new:styles/util';
 
-.legend .labels {
-    &.compact {
-        display: flex;
-        align-items: center;
-    }
-
-    :global(.label) {
-        color: white;
+.legend {
+    display: flex;
+    justify-content: space-between;
 
+    .labels {
         &.compact {
-            padding: 0.5em;
+            display: flex;
+            align-items: center;
+        }
+
+        :global(.label) {
+            color: white;
 
-            &+ .text {
-                padding-left: 0.25em;
+            &.compact {
+                padding: 0.5em;
+
+                &+ .text {
```

Comment:
`+ .text` since I don't think there's a difference between `foo+ bar` and `foo + bar` in css